src/core/mod.rs

```diff
@@ -50,8 +50,8 @@ pub use stacks_common::consts::{CHAIN_ID_MAINNET, CHAIN_ID_TESTNET, STACKS_EPOCH
 // fourth byte == highest epoch supported by this node
 // - 0x05 for 2.05
 // - 0x06 for 2.1
-pub const PEER_VERSION_MAINNET: u32 = 0x18000007;
-pub const PEER_VERSION_TESTNET: u32 = 0xfacade07;
+pub const PEER_VERSION_MAINNET: u32 = 0x18000008;
+pub const PEER_VERSION_TESTNET: u32 = 0xfacade08;
```

Comment:
Let's save that for 2.4, I'd prefer to minimize the changes here as much as possible.

Reply:
I added this commit to 2.4: https://github.com/stacks-network/stacks-blockchain/commit/3ca4b015898a82c8e960ffaedd57ac5e2c304dff

It adds a `PEER_NETWORK_EPOCH` const, and also validates it during the epochs validation, so that things like the integration tests (and also testing via genesis syncs, etc.) will panic if the network epoch isn't updated alongside the epochs list.